templates/frontend/deployment.yaml

```diff
@@ -26,6 +26,9 @@ spec:
         - name: frontend
           image: "{{ .Values.image.project }}/{{ .Values.frontend.image.repository }}:{{ .Values.frontend.image.tag }}"
           imagePullPolicy: {{ .Values.frontend.image.pullPolicy | default .Values.image.pullPolicy }}
+          env:
```

Comment:
Have you tried to add that env to dockerfile here: https://github.com/IN-CORE/incore-ui/blob/develop/Dockerfile#L10. I didn't see it in your WIP PR.

If you have already tried, then maybe the env variable can only be set at build time, not run time.

Reply:
I don't think that it needs to be there. I've checked that the docker container already have the env variable and also you can see it in the LENS. Other containers, like data service or others don't have those lines in the Dockerfile but being controlled by helm deployment

Reply:
I just tried to add it but it didn't work either.... hmm

Reply:
hmmm I searched some, but it is said that it should work with run time setting, but apparently, it doesn't work, so it should be in the build process.